### ISSUE TITLE
addon not starting

### ISSUE THREAD

**Issue by Breakerz** [v3rm0n/addon-shairport-sync#1]
I got this error on HassOS, running on Proxmox.

```
WARNING: Failed to contact D-Bus daemon.
avahi-daemon 0.7 exiting.
Found user 'avahi' (UID 86) and group 'avahi' (GID 86).
Successfully dropped root privileges.
avahi-daemon 0.7 starting up.
WARNING: No NSS support for mDNS detected, consider installing nss-mdns!
dbus_bus_get_private(): Failed to connect to socket /var/run/dbus/system_bus_socket: Permission denied
WARNING: Failed to contact D-Bus daemon.
avahi-daemon 0.7 exiting.
```

**Comment by spartanunitato** [comment]
@v3rm0n does this add-on work with the rpi 4?

**Comment by v3rm0n** [comment]
Not sure, haven't checked. I have one waiting for me to upgrade my Hassio to rpi4 so I guess I'll try it out today

**Comment by spartanunitato** [comment]
@v3rm0n I can select my pi 4 in the airplay devices menu, but there is no audio output from my pi. I keep getting this error message. I am on home assistant 0.108.9.

`ALSA lib dlmisc.c:287:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/alsa-lib/libasound_module_pcm_pulse.so ((null): Error loading shared library /usr/lib/alsa-lib/libasound_module_pcm_pulse.so: No such file or directory)`

**Comment by OXOTH1K** [comment]
Same problem on Hass.io 0.112.4 and arch  x86_64. 
On logs:
[FATAL tini (6)] exec /init failed: Exec format error

**Comment by adymanis** [comment]
Had same issues when I tried using this add-on. Fixed in PR: https://github.com/v3rm0n/addon-shairport-sync/pull/3

**Comment by v3rm0n** [comment]
Fixed in 0.0.3
